client/stylesheets/_responsive.scss

```diff
@@ -115,31 +184,35 @@
     	display: none !important;
     }
 
-    // .searchResultsCardLi {
-    // 	width: 95% !important;
-    // 	margin: 10px 0 10px 0 !important;
-    // }
+    .searchResultsCards {
+        margin-top: 0px !important;
+    }
 
-    // .searchResultsCards {
-    // 	float: none !important;
-    // 	margin-top: 0 !important;
-    // 	width: 90%;
-    // }
+    .searchResultsCardLi {
+    	width: 93% !important;
+    	margin: 10px auto !important;
+    }
 
-    // .searchResultsCardImg {
-    // 	height: 165px !important;
-    // }
-    // .searchResultsCardPrice {
-    // 	margin: 8px 0 0 0 !important;
-    // 	font-size: 0.95rem !important;
-    // }
-    // .searchResultsCardSize, .searchResultsCardCondition, .searchResultsCardLocation {
-    // 	margin: 5px 0 !important;
-    // 	font-size: 0.95rem !important;
-    // } 
-    // .searchResultsCardName {
-    // 	font-size: 0.9rem !important;
-    // }
+    .searchResultsCards {
+    	float: none !important;
+    	margin-top: 0 !important;
+    	width: 95.5%;
+    }
+
+    .searchResultsCardImg {
+    	height: 185px !important;
+    }
+    .searchResultsCardPrice {
+    	margin: 8px 0 10px 0 !important;
+    	font-size: 1rem !important;
+    }
+    .searchResultsCardSize, .searchResultsCardCondition, .searchResultsCardLocation {
+    	margin: 12px 0 !important;
+    	font-size: 1rem !important;
+    } 
+    .searchResultsCardName {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardName` should be written in lowercase with hyphens